### PR TITLE
[author][new] embed.js @ 3.6.1

### PR DESCRIPTION
**Pull request by ritz078**
The library source can be found at https://github.com/ritz078/embed.js

### CODE REVIEW

File: ajax/libs/embed-js/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      "email": "jesusprubio@gmail.com",
+      "url": "https://github.com/jesusprubio"
+    }
+  ],
+  "license": "MIT",
+  "config": {
+    "commitizen": {
+      "path": "node_modules/cz-conventional-changelog"
+    }
+  },
```
Comment (PeterDaveHello) [inline ajax/libs/embed-js/package.json]: Please remove `"config" section`

File: ajax/libs/embed-js/package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "npmName": "embed-js",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "/assets"
```
Comment (PeterDaveHello) [inline ajax/libs/embed-js/package.json]: Should be `"embed.min.js"`

File: ajax/libs/embed-js/package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    "jquery-embed"
+  ],
+  "main": "./dist/embed.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ritz078/embed.js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ritz078/embed.js/issues"
+  },
```
Comment (PeterDaveHello) [inline ajax/libs/embed-js/package.json]: Please remove `"bugs"` section

File: ajax/libs/embed-js/package.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  "keywords": [
+    "JavaScript-plugin",
+    "embed",
+    "plugins",
+    "JavaScript",
+    "embed-js",
+    "emoticon",
+    "jquery-embed"
+  ],
+  "main": "./dist/embed.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/embed-js/package.json]: Please remove `"main"` section

**Comment by ritz078** [comment]
@PeterDaveHello any updates ?

File: ajax/libs/embed-js/package.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    "name": "Ritesh Kumar",
+    "email": "rkritesh078@gmail.com",
+    "url": "https://github.com/ritz078"
+  },
+  "homepage": "http://riteshkr.com/embed.js",
+  "license": "MIT",
+  "npmName": "embed-js",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
```
Comment (PeterDaveHello) [inline ajax/libs/embed-js/package.json]: I didn't see `dist` folder under npm package:
```sh
$ ls
assets/  bower_components/  CHANGELOG.md  package.json  README.md  src/
```

**Comment by PeterDaveHello** [comment]
Maybe you need to transfer to use git auto-update, https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md

**Comment by PeterDaveHello** [comment]
Once you fixed it, please squash/fixup these commits into only one commit, and rebase this branch(importer-embed-js-3.6.10) onto our latest master branch, and then ping me again, thank you.

**Comment by PeterDaveHello** [comment]
hey @ritz078, do you need me to add this lib for you?

**Comment by ritz078** [comment]
It has been added by PR #7111

**Comment by PeterDaveHello** [comment]
Oh great, sorry that I didn't notice that, too many works here, thanks for your patience to wait, hope you enjoy it!

**Comment by ritz078** [comment]
Thanks a lot @PeterDaveHello

**Comment by PeterDaveHello** [comment]
You're welcome!

**Pull request closed without merging**